server/chroma/src/routes/authorization.rs

```diff
@@ -38,6 +38,23 @@ impl FromRequest for Authorization {
                     trace!("Value of authorization header could not be converter to UTF-8 String")
                 })?;
 
+            // Check if we're dealing with a service token
+            if authorization.starts_with("Service ") {
+                let token = authorization.chars().skip(8).collect::<String>();
+                if token.is_empty() {
+                    return Err(AuthorizationError::InvalidServiceToken);
+                }
+
+                return if data.config.service_tokens().contains(&token.as_str()) {
+                    Ok(Self {
+                        is_admin: true,
+                        user_id: 0,
```

Comment:
Are we positive that `user_id` 0 always belongs to an admin?

Reply:
Currently I've just chosen this at-random. The user ID in Chroma matches the user ID in Koala. Does Koala have an admin ID or something?

If not, or if 0 is already used by Koala, I'll think of something else. Probably turning it into an enum `User`:
```
enum User {
    KoalaUser(koala_id),
    ServiceUser(service_token),
}
```
It isn't used anywhere _yet_, so it's easy to replace.